crates/node/src/interactive_cli/context.rs

```diff
@@ -740,7 +740,7 @@ async fn handle_proposals_command(
 
             if let Some(proposal) = proposal {
                 let approvers = proxy_client.get_proposal_approvers(&proposal_id).await?;
-
+                let approvers_vec: Vec<_> = approvers.into_iter().collect();
                 let num_approvals = proxy_client.proposal_approvals(&proposal_id).await?;
```

Comment:
these two should be the same, `proposal_approvals == approvers.collect().len()`, right @alenmestrov?